### PR TITLE
Add autoscaling load balancer methods

### PR DESCRIPTION
**Pull request by captainkerk**
This adds three methods to the autoscaling service: attach_load_balancers, describe_load_balancers, and detach_load_balancers; all of which manipulate "classic" Elastic load balancers and their association to an autoscaling group.

### CODE REVIEW

File: moto/autoscaling/models.py [hunk 1]
```diff
@@ -496,6 +496,24 @@ def create_or_update_tags(self, tags):
 
             group.tags = new_tags
 
+    def attach_load_balancers(self, group_name, load_balancer_names):
```
Comment (terrycain) [inline moto/autoscaling/models.py]: Once #1255 gets merged, it wouldn't be a bad idea to see if [these](https://github.com/terrycain/moto/blob/19b75770910f2a42158044da9c69ff3783e14a89/moto/awslambda/responses.py#L36-L37) decorators would work with these responses as then you can do things like in [this](https://github.com/terrycain/moto/blob/19b75770910f2a42158044da9c69ff3783e14a89/moto/sqs/responses.py#L287) template which autogenerated a request id (useful for uniqueness when X-ray is being used).

Comment (captainkerk) [reply moto/autoscaling/models.py]: Woah, that's pretty slick. I'm definitely willing to roll that in once it's merged. Looks like that should be soon?

Comment (JackDanger) [reply moto/autoscaling/models.py]: I've just merged #1255 - let's make it happen :)

Comment (captainkerk) [reply moto/autoscaling/models.py]: On it.

Comment (captainkerk) [reply moto/autoscaling/models.py]: Done.

Comment (captainkerk) [reply moto/autoscaling/models.py]: Done. I also added it to the methods I added in #1264

**Comment by JackDanger** [comment]
This is so slick. Merging immediately. @terrycain I'm a big fan of your new decorators! @captainkerk I'm a big fan of you!

**Comment by JackDanger** [comment]
This is released as Moto 1.1.23

**Pull request merged**